CONTRIBUTING.md

```diff
@@ -94,6 +97,15 @@ make test
 make build
 ```
 
+To build the user docs locally, please refer to the below steps:
+
+```bash
+# Requires python3 and pip3 installed in your local system
+make install-site-deps
```

Comment:
@sayboras is this step needed when deploying the site? If so I think we need to make sure this is called from `make build-pages` so it's run on netlify too before deploying the site

Reply:
No, it's in the netlify settings. We define the docs directory there and the build step. It looks at runtime.txt and requirements.txt in there (https://docs.netlify.com/configure-builds/manage-dependencies/#python).

Reply:
@martina-if oh I didn't have access to netlify to confirm, but I think @dholbach comment helps to answer your question.

Related changs mainly for running and bringing up the pages locally to make sure look and feel as expected.